missing_array:
  hero:
    - [ 25.98342, 14.02023 ]
    - [ 17.18714, 1.158 ]
  engineer:
    - [ 19.32496, 4.03761 ]
    - [ 14.81307, 8.22277 ]
    - [ 26.17995, 11.67731 ]
  robot3:
    - [ 19.50331, 13.04911 ] # угд
    - [ 25.98342, 14.02023 ]
  robot4:
    - [ 19.50331, 13.04911 ]
    - [ 25.98342, 14.02023 ]
  robot5:
#    - [ 19.50331, 13.04911 ]
    - [ 25.98342, 14.02023 ]
  sentry:
    - [ 22.53553, 7.2 ]
    - [ 12.31114, 6.34838 ]

base_attacked_missing_array:
  index: [ 0, 2, 3, 4 ]
  data:
    - [ 3.587, 7.500 ]
    - [ 1.54949, 9.47109 ]
    - [ 1.54949, 5.56491 ]

Hero_BR3_attacked_missing_array:
  index: [ 2, 3, 4 ]
  data:
    - [ 5.47276, 12.60649 ]
    - [ 4.47276, 11.60649 ]
    - [ 4.47276, 13.60649 ]

